tensorflow/core/kernels/control_flow_ops.cc

```diff
@@ -80,21 +74,21 @@ REGISTER_KERNEL_BUILDER(
                               .TypeConstraint<type>("T"), \
                           SwitchOp)
 
-#define REGISTER_GPU_SWITCH(type)                         \
+#define REGISTER_DEFAULT_SWITCH(type)                     \
```

Comment:
"DEFAULT" here is a bit ambiguous (don't know "default" of what). Maybe expand to "DEVICE_DEFAULT"?
```suggestion
#define REGISTER_DEVICE_DEFAULT_SWITCH(type)                     \
```